2017/fall/syllabus/yale.adoc

```diff
@@ -143,7 +142,7 @@ For students who avail themselves of the course's other resources but still find
 
 == Problem Sets
 
-Nine problem sets are assigned during the semester. Late work is not ordinarily accepted, except in cases of emergency. 
+Nine problem sets are assigned during the semester. Late work is not ordinarily accepted, except with a Dean's excuse.  (Graduate students should e-mail heads@cs50 as early as possible to request an extension.)
```

Comment:
Best to write in full as `heads@cs50.yale.edu`, since it's not mentioned elsewhere? (Spam filters are good.)